packages/analytics-js-plugins/src/utilities/consentManagement.ts

```
@@ -0,0 +1,63 @@
+    const allowedConsentIds = state.consents.data.value.allowedConsentIds as string[];
+    const matchPredicate = (consent: string) => allowedConsentIds.includes(consent);
+
+    // match the configured consents with user provided consents as per
+    // the configured resolution strategy
+    const resolutionStrategy =
+      cmpConfig.resolutionStrategy ?? state.consents.resolutionStrategy.value;
+    switch (resolutionStrategy) {
+      case 'any':
+      case 'or': // Deprecated
+        return configuredConsents.some(matchPredicate) || configuredConsents.length === 0;
+      case 'all':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)